dired-rsync.el

```diff
@@ -71,14 +71,9 @@
 
 ;; Helpers
 
-(defun dired-rsync--is-remote-tramp-p (file-or-path)
-  "Return non-nil if FILE-OR-PATH is remote."
-  (or (string-prefix-p "/scp:" file-or-path)
-      (string-prefix-p "/ssh:" file-or-path)))
-
 (defun dired-rsync--quote-and-maybe-convert-from-tramp (file-or-path)
   "Reformat a tramp FILE-OR-PATH to one usable for rsync."
-  (if (dired-rsync--is-remote-tramp-p file-or-path)
+  (if (tramp-tramp-file-p file-or-path)
```

Comment:
Won't this fail if the tramp remote doesn't also have ssh setup which rsync can use?

Reply:
We probably can just check the exit code of rsync?
I can do it if you want but I'm busy the next 1-2 weeks
so it would take a while.

Imho you can still merge this PR now as it doesn't
break anything *more* than what's not already broken ;)
It just allows `sudo:...` prefix (instead of `ssh:root@...`) to work too.